src/common/access_pattern.c

```diff
@@ -350,11 +352,15 @@ xdd_save_seek_list(target_data_t *tdp) {
 		}
 		average = (uint64_t)(total / sp->seek_total_ops);
 		/* Print the seek list into the specified file */
+		fprintf(tmp, "# Block size=%d, Request size=%d, Numreqs=%ld\n",
+                (int32_t)tdp->td_block_size,
+                (int32_t)tdp->td_reqsize,
+                (long int)tdp->td_numreqs);
 		fprintf(tmp,"# Longest seek=%llu, Shortest seek=%llu, Average seek distance=%llu\n",
```

Comment:
Just put the number of requests in this line of the head. Stick with the same format as the seek stuff:
`, Number of Request=%ld`